examples/platforms/nrf528xx/src/radio.c

```diff
@@ -488,13 +489,27 @@ otError otPlatRadioReceive(otInstance *aInstance, uint8_t aChannel)
     }
 
     nrf_802154_tx_power_set(GetTransmitPowerForChannel(aChannel));
-
     result = nrf_802154_receive();
     clearPendingEvents();
 
     return result ? OT_ERROR_NONE : OT_ERROR_INVALID_STATE;
 }
 
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE && OPENTHREAD_CONFIG_MAC_SOFTWARE_RX_TIMING_ENABLE
```

Comment:
```suggestion
#if OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2
```
`otPlatRadioGetCaps` indicates `OT_RADIO_CAPS_RECEIVE_TIMING` for OT1.2, then this method should be provided for OT1.2.